d3/d3.d.ts

```diff
@@ -119,7 +119,7 @@ declare module D3 {
         * @param arr Array to search
         * @param map Accsessor function
         */
-        extent<T, U>(arr: T[], map: (v: T) => U): U[];
+        extent<T, U>(arr: T[], map: (v?: T, i?: number) => U): U[];
```

Comment:
Yes I will modify all the signatures of map function later. And I think the third parameter is needless in the signatures, according to the source code. map.call(this,v,i) --> this.map(v,i)